rskj-core/src/main/java/co/rsk/config/BridgeMainNetConstants.java

```diff
@@ -144,6 +144,8 @@
 
         minSecondsBetweenCallsReceiveHeader = 300;  // 5 minutes in Seconds
         maxDepthBlockchainAccepted = 25;
+
+        percentageAboveFeeForPegouts = 20;
```

Comment:
What's the meaning of this value? Sounds like if you need to pay 100 in fee you need to pegout at least 120, while the reasonable for me is that if you need to pay 100 in fees, the minimum pegout should be about 500, so the user doesn't lose that much

Reply:
The value will be updated accordingly before code freeze, I just defined a "mockup" value. I agree with you that 20% sounds to low